LayoutTests/fast/events/mouse-events-on-node-deletion.html

```diff
@@ -0,0 +1,86 @@
+    <!DOCTYPE HTML>
```

Comment:
What's up with the indentation?
We don't normally indent the doc type like this.